### PR TITLE
doc: Add information on adoptium server backups

### PR DESCRIPTION
**Pull request by sxa**
Fixes https://github.com/adoptium/infrastructure/issues/1295

For ease of reviewing feel free to start browsing the formatted pages at https://github.com/sxa/infrastructure/tree/backupdocs#backups

Note that the script being added here is the one that has been used live on the server for sometime other than me changing backticks to `$(...)` which is a known thing that upsets the linter so the functionality is basically sound ;-)

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [x] other documentation is changed or added (if applicable)

### CODE REVIEW

File: tools/run_backups [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+set -x
+cd /home/backups || exit 1
+echo $(date +%T) : Starting backups of jenkins config.xml files ...
+ssh jenkins@ci.adoptium.net 'ls -1 /home/jenkins/.jenkins/plugins/*.?pi | while read FILENAME; do basename $FILENAME ; unzip -p $FILENAME META-INF/MANIFEST.MF | sort -d | grep "Short-Name\|Long-Name\|Plugin-Version" ; echo --- ; done > /home/jenkins/.jenkins/pluginVersions.txt'
+#ssh jenkins@ci.adoptopenjdk.net 'tar czf - $(find . -name config.xml )' > jenkins/jenkins.$(date +%Y%m%d).tar.gz
```
Comment (karianna) [inline tools/run_backups:6]: commented out code?

**Review by karianna — CHANGES_REQUESTED** [review]
commented out code in the shell script

File: tools/run_backups (hunk 1 above)
Comment (sxa) [reply tools/run_backups]: That was due to a very recent change where I wanted to keep the older version, but happy to remove it.

**Review by karianna — APPROVED** [review]

**Comment by tellison** [comment]
That readme file is quite out of date

**Comment by sxa** [comment]
@tellison are you happy to approve this now that a new issue has been created to cover updating of the document generally?

**Pull request merged**